src/engine-scripts/puppet/scroll.js

```diff
@@ -1,6 +1,6 @@
 module.exports = async ( page ) => {
 	await page.evaluate( () => {
-		window.scroll( 0, window.innerHeight * 2 );
+		window.scroll( 0, 1500 );
```

Comment:
the old value was

```suggestion
		window.scroll( 0, window.innerHeight );
```
do we want to revert it to the old value or 1500 is fine